applications/tari_indexer/src/json_rpc/server.rs

```diff
@@ -71,6 +71,8 @@ async fn handler(Extension(handlers): Extension<Arc<JsonRpcHandlers>>, value: Js
         "get_comms_stats" => handlers.get_comms_stats(value).await,
         "list_substates" => handlers.list_substates(value).await,
         "get_substate" => handlers.get_substate(value).await,
+        "get_substates" => handlers.get_substates(value).await,
+        "fetch_substates" => handlers.get_substates(value).await,
```

Comment:
The `get_susbtates` handler is not implemented. I can imagine a user asking what the difference is between `list_substates` and `get_substates`. 

```suggestion
        "get_substates" | "fetch_substates" => handlers.get_substates(value).await,
```